packages/matter-node.js/test/cluster/GeneralCommissioningServerTest.ts

```diff
@@ -28,8 +28,6 @@ import { SecureSession } from "@project-chip/matter.js/session";
 import * as assert from "assert";
 import { callCommandOnClusterServer, createTestSessionWithFabric } from "./ClusterServerTestingUtil";
 
-/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
this can be removed because?

Reply:
New eslint warned that it was unnecessary.  I'm not sure why it was there but I assume it was a lint bug that got fixed.

Reply:
interesting ... ok ... In fact yes we do such null assertations in the code and did not have disabled the check ... soo I just wonder if we loose anything by other issues right now?

Reply:
I added comments detailing the issue.  Resolving for now per conversation in Discord